studio/pages/project/[ref]/auth/settings.tsx

```diff
@@ -67,6 +67,16 @@ const Settings = () => {
   }, [config])
 
   const onFormSubmit = async (model: any) => {
+    model.SECURITY_CAPTCHA_PROVIDER = 'hcaptcha'
+
+    if (!model.SECURITY_CAPTCHA_SECRET) {
```

Comment:
this should be `SECURITY_CAPTCHA_PROVIDER_SECRET`

Reply:
TBH it's a bit confusing, and fixing it will require a migration on infra. I'm not sure it's worth it though at this time.

The reason is that at some point this was registered in infra as `SECURITY_CAPTCHA_SECRET` as the GoTrue custom schema, while in GoTrue it is [`SECURITY_CAPTCHA_PROVIDER_SECRET`](https://github.com/supabase/gotrue/blob/397c949443365e4ebf5ddd8bd98a64eecd4fa9b4/conf/configuration.go#L179). The infra worker will pick it up correctly as the mapping is done between the variations.

I've tagged you on the important line in that repo.